### DISCUSSION TITLE
Welcome to ERCF_v2 Discussions!

### DISCUSSION THREAD

**Discussion by Miriax** [Enraged-Rabbit-Community/ERCF_v2#81] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by norris108** [comment]
hi everyone!

looks like i'm the first to comment here! 😁

i'm in the process of building and configuring my ercf v2

it's been a slow process, of my own making, as i've been using my voron 2.4 in between testing and configuration of ercf v2 for other projects

i'm hopeful and looking forward to having ercf v2 up-and-running very soon!

**Comment by chispolas** [comment]
Hello! 
Does ERCF v2 works with another Toolhead instead the SteathBurner? I would like install it in my voron trident but i have a XOL2 Toolhead....
=)

**Reply by norris108** [reply]
In theory, ERCF will work with _any_ toolhead

See [here](https://github.com/SouthAsh1/Xol-metrix) for a XOL version

**Reply by chispolas** [reply]
thank you!! i will see this link =)

**Reply by norris108** [reply]
I'd also recommend joining the #ercf_questions channel on the [Voron discord](https://discord.gg/voron)

It's a very vibrant community and you'll be sure to get answers to any questions you may have

**Comment by So6Rallye** [comment]
Hi,

I have a project to build a custom IDEX printer based on Klipper with ERCF V2.

I need to know if the printer can handle 2 ERCF (one on each printer head)

I think it's technically possible but before explore more deep of this project, I want to ask the community.

Thanks and happy printing

**Comment by gorf25** [comment]
I am having trouble getting the servo to get the filiment to move i have the SAVOX SH0256MG servo but can't get it ajusted if i go a little to far down it seems to release back in a second or to if i adjust it down so its not as far and don't release back it doesn't seem far enough and will not grab the filiment any suggestions, should i try a diffrent servo?

**Reply by norris108** [reply]
as i said on discord yesterday, you should join the [ercf community discord](https://discord.gg/MPZ4cHm9gm)

**Comment by gorf25** [comment]
sorry you have to forgive me i'm in the 70's and some time have a hard 
time finding my away around the discord..
but got there no and thanks..



------ Original Message ------
From "Mark Norris" ***@***.***>
To "Enraged-Rabbit-Community/ERCF_v2" ***@***.***>
Cc "gorf25" ***@***.***>; "Comment" 
***@***.***>
Date 6/25/2025 3:38:51 AM
Subject Re: [Enraged-Rabbit-Community/ERCF_v2] Welcome to ERCF_v2 
Discussions! (Discussion #81)
